examples/pubVerifiable.example.ts

```diff
@@ -37,7 +37,12 @@ export async function publicVerifiableTokens(): Promise<void> {
     //     |<----- Request ------+                   |           |
     const redemptionContext = crypto.getRandomValues(new Uint8Array(32));
     const originInfo = ['origin.example.com', 'origin2.example.com'];
-    const tokChl = await createPrivateToken(issuer, redemptionContext, originInfo);
+    const tokChl = await PrivateToken.fromInfos(
```

Comment:
`PrivateToken.create` is a more natural name.

Reply:
updated